src/jobs/js_push_build_to_bucket_job.yml

```diff
@@ -25,9 +20,6 @@ steps:
 # Parameters
 
 parameters:
-  isopod_version:
-    type: string
-    default: ${ISOPOD_VERSION}
```

Comment:
I think if you remove that it will break jobs where this parameter is supplied. if you keep it and add a "deprecated" comment it wouldn't break those builds but still use the new image 🤔

Reply:
That's fine, I will cleanup apps which uses it. We only have one 🤞